PWGDQ/Core/VarManager.h

```diff
@@ -5215,6 +5251,107 @@ void VarManager::FillZDC(T const& zdc, float* values)
   values[kTimeZPC] = zdc.timeZPC();
 }
 
+template <typename TBCs, typename TFT0s, typename TFDDs, typename TFV0As>
+void VarManager::FillFIT(uint64_t midbc, std::vector<std::pair<uint64_t, int64_t>>& bcMap, TBCs const& bcs, TFT0s const& /*ft0s*/, TFDDs const& /*fdds*/, TFV0As const& /*fv0as*/, float* values)
+{
+  if (!values) {
+    values = fgValues;
+  }
```

Comment:
In this function you do not cover the case when the FIT data is in the reduced DQ data model and is being used from the analysis workflow (e.g. from table reader). Is this something you plan to add later or?